src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -175,17 +175,42 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
+    public static String getIndividualPreferredHearingChannel(String hearingType,
+                                                                        HearingSubtype hearingSubtype,
+                                                                        HearingOptions hearingOptions) {
+        if (eitherNull(hearingType, hearingSubtype)) {
+            throw new IllegalStateException("hearingType and/or hearingSubtype null");
+        }
```

Comment:
This shouldn't be here.
HearingType will be null for OtherParty. 
HearingType and hearingSubtype will both be null for JointParty

Reply:
Being addressed here: **https://github.com/hmcts/sscs-hearings-api/pull/185**